wikipedia_cleanup/data_filter.py

```diff
@@ -86,6 +86,75 @@ def __str__(self) -> str:
         )
 
 
+class DiscardAttributesDataFilter(AbstractDataFilter):
+    def _filter_for_property(self, changes: List[InfoboxChange]) -> List[InfoboxChange]:
+        raise NotImplementedError("This method should never be called.")
+
+    def filter(
+        self, changes: List[InfoboxChange], initial_num_changes: int
+    ) -> List[InfoboxChange]:
+        self.filter_stats.initial_num_changes = initial_num_changes
+        self.filter_stats.input_num_changes = initial_num_changes
+        self.filter_stats.output_num_changes = initial_num_changes
+
+        sparse_changes = []
+        for change in changes:
+            sparse_change: InfoboxChange = SparseInfoboxChange()  # type: ignore
```

Comment:
This typing error makes sense and should probably not be ignored. We can't tell the user that we give them `InfoboxChange` but then they cannot access all the attributes of it as its actually a `SparseInfoboxChange`. It also no longer has a tostring method

Reply:
But that's the whole point of it. There is no real way to use a SparseInfoboxChange without ignoring that error in the pipeline, we would need to always add checks in each other filter method in order to make it work. Thats the risk when using that kind of filter